### PR TITLE
contributing

### PR DESCRIPTION
**Pull request by kp-cat**
### Describe the purpose of your pull request

CONTRIBUTING.md has been added.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+```bash
+mix local.rebar --force
+mix local.hex --force
+mix deps.get
+```
+
+## Running tests
+
+```bash
+mix coveralls.json
```
Comment (randycoulman) [inline CONTRIBUTING.md]: For running tests locally, people might prefer to use `mix test` or `mix test.watch` rather than `mix coveralls.json`.

**Review by randycoulman — APPROVED** [review]
One optional suggestion below, but otherwise this looks great!

**Comment by randycoulman** [comment]
@kp-cat Is this ready to merge? I'd like to get your README me change in before I release v1.0.2 with the Elixir 1.11 fixes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kp-cat) [reply CONTRIBUTING.md]: thanks I've updated the PR with `mix test`.

**Comment by kp-cat** [comment]
I've merged it. Thanks for for your review

**Pull request merged**